docs/getting-started-guides/docker-multinode/master.sh

```diff
@@ -95,16 +96,17 @@ detect_lsb() {
 
 
 # Start the bootstrap daemon
+# TODO: do not start docker-bootstrap if it's already running
 bootstrap_daemon() {
     sudo -b docker -d \
-    -H unix:///var/run/docker-bootstrap.sock \
-    -p /var/run/docker-bootstrap.pid \
-    --iptables=false \
-    --ip-masq=false \
-    --bridge=none \
-    --graph=/var/lib/docker-bootstrap \
-    2> /var/log/docker-bootstrap.log \
-    1> /dev/null
+        -H unix:///var/run/docker-bootstrap.sock \
+        -p /var/run/docker-bootstrap.pid \
+        --iptables=false \
+        --ip-masq=false \
+        --bridge=none \
+        --graph=/var/lib/docker-bootstrap \
+            2> /var/log/docker-bootstrap.log \
+            1> /dev/null
```

Comment:
@fgrzadkowski Do you think I could replace `sudo -b` with `&` at the end?
I would like to remove all `sudo`s, as they don't exist on all platforms and is unnecessary since we already are `root`